public/data/food/processes.json

```diff
@@ -490,25 +490,25 @@
     "comment": "",
     "database": "Ecobalyse",
     "impacts": {
-      "acd": 0.3288452922,
-      "ozd": 3.971567038e-7,
-      "cch": 22.26322646,
-      "fwe": 0.0009156175281,
-      "swe": 0.05607279472,
-      "tre": 1.466683729,
-      "pco": 0.03400605395,
-      "pma": 2.190875281e-6,
-      "ior": 1.134379763,
-      "fru": 49.53803992,
-      "mru": 2.175375607e-5,
-      "ldu": 1859.186099,
-      "wtu": 2.614027352,
-      "htc": 9.366607136e-9,
-      "htn": 2.669635091e-7,
-      "etf": 29.703249436,
-      "etf-c": 31.790270159640002,
-      "htc-c": 1.9641705902e-9,
-      "htn-c": 6.402503876e-8
+      "acd": 0.2342054931,
+      "ozd": 2.927223966e-7,
+      "cch": 16.36861768,
```

Comment:
no idea why these values changed. Any idea @ccomb ?

Reply:
I'm unsure but if any doubt we can relaunch the export